app/client/components/filterModule/FilterModule.jsx

```diff
@@ -50,15 +54,21 @@ class FilterModule extends React.Component {
     };
 
     this.props.loadAllowedFilters(this.props.filterObject);
-
-    console.log("Default groupby is", props.defaultGroupBy);
   }
 
   componentDidMount() {
     this.loadMetricsWithParams();
+
+    let login = this.props.login;
+    let channel_name = window.PUSHER_ENV_CHANNEL + "-" + login.organisationId;
+    let channel = getChannel(channel_name);
+    if (channel) {
+      channel.bind(this.props.filterObject, () => this.loadMetricsWithParams());
+    }
   }
 
   componentDidUpdate(prevProps, prevState, snapshot) {
+    //Refresh Data if the date range changes or if the transfer list changes (normally from pusher)
```

Comment:
Could this potentially cause issues on busy instances? If there are several transactions per minute, that's a lot of hits to the metrics dash. Perhaps it would help to have a cool-down time

Reply:
Nick to add 10 seconds min refresh time